azure-bootstrap.yml

```
@@ -0,0 +1,45 @@
+      type: github
+      name: nanoframework/nf-tools
+      endpoint: nanoframework
+
+jobs:
+- job: Trigger
+  displayName: Trigger Azure Dev Ops build and test pipeline
+  pool:
+    vmImage: 'ubuntu-latest'  
+
+  steps:
+  - template: azure-pipelines-templates/device-bootstrap.yml@templates
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)